packages/holocron/src/holocronModule.jsx

```diff
@@ -94,7 +94,7 @@ export default function holocronModule({
   shouldModuleReload,
   loadModuleData,
   mergeProps,
-  options = {},
+  options = { provideModuleState: true },
```

Comment:
This is breaking to anyone passing any options today. If a user is passing any options at all this default value will not be present.

Reply:
addressed b9ed653